graph/path/shortest.go

```diff
@@ -238,6 +249,9 @@ var (
 	defacedBits = math.Float64bits(defaced)
 )
 
+// newAllShortest returns a all-pairs shortest path forrest for paths with the
```

Comment:
"forrest"?

Reply:
Full of trrees.

Reply:
That's a thing in graph theory?

Reply:
The typo is a Dan thing. I have corrected it.

Reply:
Sorry, missed the joke :D